### DISCUSSION TITLE
第4节课作业(6班)

### DISCUSSION THREAD

**Discussion by maxchiron** [InternLM/Tutorial#275] [作业]
第4节课作业(6班)

**Comment by zhangzefeng92** [comment]
微信名：捂上眼睛看世界
![图片](https://github.com/InternLM/tutorial/assets/58167242/63fa0401-7b0e-4987-9923-3528fde25d8c)

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/e174b8da-807d-43bd-9414-0cd57682faa1](https://app.yinxiang.com/fx/e174b8da-807d-43bd-9414-0cd57682faa1)

**Comment by JackieGLQ** [comment]
微信名:老高
https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#HPrtdx5CpozIUHx383vchyaPnvg

**Comment by RalphGuo** [comment]
guos-https://gitee.com/RalphGs/InternLM-assignment/tree/master/%E8%AF%BE%E7%A8%8B4
